EAO/UI/Inspections/ProjectListController.swift

```diff
@@ -142,6 +153,7 @@ extension ProjectListController: UITableViewDelegate, UITableViewDataSource{
 	
 	func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
 		result?(filtered?[indexPath.row].string ?? projects?[indexPath.row])
-		pop()
+		popViewController()
```

Comment:
Unwind

Reply:
In this an all other cases below all the information is either sent back via callbacks or saved into realm before transitions. And since unwind segue is going to be called from code, I think that using it adds extra code and complexity without any benefits